MonoGame.Framework/Color.cs

```diff
@@ -1761,6 +1761,12 @@ public static Color Multiply(Color value, float scale)
             return new Color((int)(value.R * scale), (int)(value.G * scale), (int)(value.B * scale), (int)(value.A * scale));
         }
 
+	    /// <summary>
+        /// Multiply <see cref="Color"/> by value.
+        /// </summary>
+        /// <param name="scale">Multiplicator.</param>

```

Comment:
I don't think the periods here are necessary unless you are writing a full sentence. 

That said we probably want to define some guidance here to keep consistency for this @SimonDarksideJ @dellis1972 ? 

Maybe we should have some "examples" of different types of docs in the documentation contributing guide?  Maybe like the coding standard showing right and wrong way to write various types of API docs?

Reply:
@tomspilman sorry I updated the PR before I saw this comment.

I took the existing API XML from the multiply operator above and just reused it since it's the same thing, just swapped params.

The commit I just submitted has, I feel, a better Api documentation of the methods.  But that's personal bias 😄.  If you could look at that.

But yes having a specific guideline would be great.  The one in the coding style doc is really basic, and myself and @Maniekko are using the XNA docs @dellis1972 provided to fill in the missing holes where we can